panel/widgets/chat.py

```diff
@@ -211,7 +215,7 @@ class ChatEntry(CompositeWidget):
     show_timestamp : bool
         Whether to display the timestamp of the message.
     """
-    _ignored_refs: ClassVar[Tuple[str,...]] = ('value',)
+    _ignored_refs: ClassVar[List[str]] = ['value']
```

Comment:
I can see the value of having a class variable as a non-mutable object.

Reply:
I totally agree. I previously had a pr to correct this in the entire codebase. But it never came in. Dont know the reason.

Reply:
I think we should indeed prefer `Tuple` everywhere.

Reply:
I've changed to `Tuple` everywhere and also fixed for `_linkable_properties` properties. But the scope here is `chat.py` so I don't plan to do more in this PR.